01-JS-DOM-APIs/1/style.css

```diff
@@ -0,0 +1,7 @@
+section {
```

Comment:
In real projects we should not use tagname selectors, because it would add the defined rules to all related elements in our htmls, instead of doing this we use classes, so that attaching the class to elements that we want to be affected with these style changes are affected, but the others keep their native aspect.